src/styles/blocks/grid.scss

```diff
@@ -0,0 +1,21 @@
+.grid-container {
+  width: 100%;
```

Comment:
this property makes failed some tests.
for width more than 1024px it is should be 1024 px (4 * card_widht + (4 - 1) * gap(48px) + paddings).
and set margin: auto;
make flexible for other widths.
actual:
![Screenshot from 2023-10-26 12-43-50](https://github.com/mate-academy/layout_catalog/assets/33029864/12699b89-2698-4eef-a8d8-ed6cf01843df)
after fix:
![Screenshot from 2023-10-26 12-47-49](https://github.com/mate-academy/layout_catalog/assets/33029864/c8a359aa-678c-4244-bfc5-48a891b247c1)